Doc/tutorial/introduction.rst

```diff
@@ -478,73 +478,123 @@ example::
 First Steps Towards Programming
 ===============================
 
-Of course, we can use Python for more complicated tasks than adding two and two
-together.  For instance, we can write an initial sub-sequence of the
-`Fibonacci series <https://en.wikipedia.org/wiki/Fibonacci_number>`_
-as follows::
-
-   >>> # Fibonacci series:
-   ... # the sum of two elements defines the next
-   ... a, b = 0, 1
-   >>> while a < 10:
-   ...     print(a)
-   ...     a, b = b, a+b
-   ...
+We can use Python for more complex tasks than adding two and two together.
+For instance, we can compute the numbers of the
+`Fibonacci series <https://en.wikipedia.org/wiki/Fibonacci_number>`_.
+To do that we need to utilize three following concepts.
+
+Multiple Assignment
+-------------------
+
+A *multiple assignment* of variables us allows to set values of more than one
+variable on a single line.  Notice that if the value assignment is a
+expression, it is first evaluated and then assigned::
+
+   >>> a, b = 1, 5  # multiple assignment of two variables
+   >>> a
    0
```

Comment:
Needs to be `1`
![image](https://github.com/python/cpython/assets/23421705/900d91f6-44cd-46e8-9e27-b4567d91e9d8)

Reply:
Is this a doctest bug?